pkg/exchange/ftx/exchange.go

```diff
@@ -78,7 +82,10 @@ func NewExchange(key, secret string, subAccount string) *Exchange {
 		panic(err)
 	}
 
+	client := ftxapi.NewClient()
+	client.Auth(key, secret, subAccount)
```

Comment:
Just an TODO item in the future:  Change the name to `SetAuth` instead of `Auth` will be more descriptive.
Also we should move out the fields to some global struct inside `types` like `types.KeyHolder` or something.

Reply:
we can do that later~